packages/core/framework/src/http/express-loader.ts

```diff
@@ -55,11 +61,33 @@ export async function expressLoader({ app }: { app: Express }): Promise<{
   }
 
   app.set("trust proxy", 1)
-  app.use(
-    morgan("combined", {
-      skip: () => isTest,
-    })
-  )
+
+  /**
+   * Method to skip logging HTTP requests. We skip in test environment
+   * and also exclude files served by vite during development
+   */
+  function shouldSkipHttpLog(req: MedusaRequest, res: MedusaResponse) {
+    if (isTest) {
+      return true
+    }
+    if (NOISY_ENDPOINTS_CHUNKS.some((chunk) => req.url.includes(chunk))) {
+      return true
+    }
+    return false
```

Comment:
```suggestion
    const isNoisy = NOISY_ENDPOINTS_CHUNKS.some((chunk) => req.url.includes(chunk))
    return isTest || isNoisy
```

if we use the regexp above it can become

```suggestion
    const isNoisy = !!req.url.match(NOISY_ENDPOINTS_CHUNKS)
    return isTest || isNoisy
```

Reply:
Shouldn't be ignore the check if we are `test` environment?

Reply:
It was returning true in both cases already but otherwise yes we might enable them in dev

Reply:
And in case I missunderstoor, here it is a short-circuit evaluation, the right hand side of the operand wont get evaluated if the left hand side is returning true